react-native-libraries.json

```diff
@@ -9823,5 +9823,18 @@
     "examples": ["https://github.com/Malaa-tech/react-native-simple-keypad/tree/main/example"],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/Alwinator/react-native-http-bridge-refurbished",
+    "npmPkg": "react-native-http-bridge-refurbished",
+    "examples": [
+      "https://github.com/Alwinator/react-native-http-bridge-refurbished#example"
+    ],
```

Comment:
```suggestion
```

We consider apps as examples, so remove the Readme link. User will land on the GitHub page by licking the package name, so they will see the Readme content anyway.